App Component Tutorial/projects/AppComponentTutorial.json

```diff
@@ -0,0 +1,322 @@
+{
+  "links" : [ {
+    "source" : "type/com.vantiq.engines.EngineSpeed",
+    "target" : "app/EngineMonitor"
+  }, {
+    "source" : "app/EngineMonitor",
+    "target" : "type/com.vantiq.engines.SystemHUD"
+  }, {
+    "source" : "type/com.vantiq.engines.EngineTemp",
+    "target" : "app/EngineMonitor"
+  }, {
+    "source" : "type/com.vantiq.engines.SystemHUD",
+    "target" : "subscription/SystemHUD"
+  }, {
+    "source" : "type/com.vantiq.engines.SystemStatus",
+    "target" : "client/EngineSimulation"
+  }, {
+    "source" : "type/com.vantiq.engines.SystemHUD",
+    "target" : "client/EngineSimulation"
+  }, {
+    "source" : "client/EngineSimulation",
+    "target" : "topic//my/response/topic"
+  }, {
+    "source" : "app/AggregateUpdates",
+    "target" : "procedure/AggregateUpdates.generate"
+  }, {
+    "source" : "type/com.vantiq.engines.SystemStatus",
+    "target" : "app/EngineMonitor"
+  }, {
+    "source" : "app/EngineMonitor",
+    "target" : "app/AggregateUpdates"
+  } ],
+  "name" : "AppComponentTutorial",
+  "options" : {
+    "description" : "The Component Tutorial demonstrates how to create a Component from scratch. The AggregateUpdates Component is a general\nrate limiter. For a given Type, inbound events are split using the Type's natural key. The most recent value for each key\nis cached and then written to the database at a maximum frequency provided by the user.\n\nThe AggregateUpdates Component is used to replace the UpdateSystemStatus task in the EngineMonitor App created in the Introductory Tutorial.\n",
```

Comment:
can you remove the 2 \n from the first paragaph.   (Would also make the first sentence its own paragraph.)
```suggestion
    "description" : "The Component Tutorial demonstrates how to create a Component from scratch. \n\nThe AggregateUpdates Component is a general rate limiter. For a given Type, inbound events are split using the Type's natural key. The most recent value for each key is cached and then written to the database at a maximum frequency provided by the user.\n\nThe AggregateUpdates Component is used to replace the UpdateSystemStatus task in the EngineMonitor App created in the Introductory Tutorial.\n",
```
Otherwise the description looks choppy when I import it:
![image](https://user-images.githubusercontent.com/27791185/113360345-73f95280-92fe-11eb-9ffb-b0f396d09790.png)